CT-1572 move libreoffice build

misaka opened the pull request.

aliuk2012 commented on app/policies/case_policy.rb:
```diff
@@ -14,4 +14,24 @@ def can_add_case?
+      if user.assigner?
+        scope.all
+      elsif user.drafter?
+        scope.select {|kase| kase.drafter == user }
```
Shouldn't we should all case which have been assigned to a particular kilo? it might cause confusion whose doing what if we dont have "whose it with" status

EdwardAndress replied:
This will show all the cases that have been assigned to the current user.

EdwardAndress replied:
Does that answer your question?

EdwardAndress replied:
I am confused

aliuk2012 replied:
Sorry you are right, I misunderstood what was happening here. A Kilo can only see their cases in the case list view. When I was manually testing it I was able to access another kilo's case by editing the url. I think we should maybe look to lock this down either as part of this PR or perhaps get it ticketed up as another piece of work

EdwardAndress replied:
Yep!  Good point.  Also, if it's not clear we should try to improve that :)

aliuk2012 replied:
No I think its clear, I just need to review more carefully ;P

misaka committed:
CT-1572 add Dockerfile.base to build base image

This base Docker image is meant to speed up our app builds, since the base image
needs to containe LibreOffice which takes a good bit of time to install with
apt.

misaka committed:
CT-1572 change FROM image for docker build to our project's base

This base image will already have libreoffice installed, so this should speed up
build times and, if we're lucky, maybe even deploy times too.

misaka committed:
CT-1572 app images are in registry.service.dsd.io

Not ministryofjustice

misaka committed:
CT-1572 remove commented-out lines from Dockerfiles
